code/modules/mob/living/silicon/silicon.dm

```diff
@@ -94,6 +94,14 @@
 	GLOB.silicon_mobs -= src
 	return ..()
 
+///Sets cyborg gender from preferences. Expects a client.
+/mob/living/silicon/proc/set_gender(client/player_client)
+	var/silicon_gender = player_client.prefs.read_preference(/datum/preference/choiced/silicon_gender)
+	if(silicon_gender == "Use gender")
```

Comment:
Make this default value a variable on `/datum/preference/choiced/silicon_gender`, like `/datum/preference/choiced/silicon_gender/var/use_character_gender = "Use character gender"`, then on every reference do `/datum/preference/choiced/silicon_gender::use_character_gender`. Avoids magic strings (without the usual define clutter)